backend/config.json.example

```diff
@@ -118,7 +118,8 @@
       "type": "vault",
       "host": "http://vault1:8200/",
       "prefix": "/v1/",
-      "backend": "secret/"
+      "backend": "secret/",
+      "ttl": 600000
```

Comment:
Fair enough - shall we tackle the change to `.js` in this PR or later in another one?